php/PHPCD.php

```diff
@@ -2,20 +2,15 @@
 namespace PHPCD;
 
 use Psr\Log\LoggerInterface as Logger;
+use Psr\Log\LoggerAwareTrait;
 use Lvht\MsgpackRpc\Server as RpcServer;
 use Lvht\MsgpackRpc\Handler as RpcHandler;
+use PHPCD\PHPFileInfo\PHPFileInfoFactory;
+use PHPCD\ClassInfo\ClassInfoFactory;
 
 class PHPCD implements RpcHandler
 {
-    const MATCH_SUBSEQUENCE = 'match_subsequence';
-    const MATCH_HEAD        = 'match_head';
-
-    private $matchType;
-
-    /**
-     * @var Logger
-     */
-    private $logger;
+    use LoggerAwareTrait;
```

Comment:
why we need the `LoggerAwareTrait`? What it only offer is the `setLogger` method which is of no use for us, because we just need to make assignment for the `$logger` property in the constructor.